concepticondata/conceptlists/Carling-2017-240c.tsv

```diff
@@ -51,7 +51,7 @@ Carling-2017-240c-49	49	646	ASH	ash(es)	12208	Physical/natural world
 Carling-2017-240c-50	50	778	SMOKE (EXHAUST)	smoke (n)	12209	Physical/natural world	
 Carling-2017-240c-51	51	221	FIRE	fire	12210	Physical/natural world	
 Carling-2017-240c-52	52	639	MOUNTAIN	mountain	12211	Physical/natural world	
-Carling-2017-240c-53	53	3310	SHORE OR BANK	shore, bank	12212	Physical/natural world	
```

Comment:
What is the reason for reducing this to just SHORE?

Reply:
SHORE is the hyperonym.

Reply:
I discontinued both SHORE OR BANK and WATERSIDE since SHORE already refers to any kind of land on the side of a body of water, no matter whether of a river, a lake or a sea/ocean.